minifi/minifi-assembly/src/main/assembly/dependencies.xml

```diff
@@ -56,6 +56,7 @@
                 <include>org.apache.nifi.minifi:minifi-framework-api</include>
                 <include>org.apache.nifi.minifi:minifi-runtime</include>
                 <include>org.apache.nifi.minifi:*:nar</include>
+                <include>org.apache.nifi:nifi-mqtt:nar</include>
```

Comment:
I think you'd need to add the same to dependencies-windows-service.xml

Reply:
Thx, fixed